toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -54,6 +54,31 @@ while read -r package || [ -n "$package" ]; do
     install_one_toolchain_rpm "$package"
 done < "$packages"
 
+# If the host machine rpm version is >= 4.16 (such as Mariner 2.0), it will create an "sqlite" rpm database backend incompatible with Mariner 1.0 (which uses "bdb")
+# To resolve this, enter the 1.0 chroot after the packages are installed, and use the older rpm tool in the chroot to re-create the database in "bdb" format.
+TEMP_DB_PATH=/temp_db
+echo "Setting up a clean RPM bdb (Berkeley DB) database in the chroot under '$TEMP_DB_PATH'."  | tee -a "$chroot_log"
+
+# These nodes are required in the chroot for certain tools (most importantly, the NSS library initialization required by rpm)
+sudo mkdir -pv $chroot_builder_folder/{dev,proc,sys,run}
```

Comment:
Do we need `proc`, `sys`, and `run`? And if so, shouldn't they be created with `mknod` as well? Or is RPM content with empty directories but will fail without them?

Reply:
I don't think these are necessary for the commands we run here in the chroot. Thanks for pointing it out - I'll remove these.